config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml

```diff
@@ -346,18 +346,26 @@ teams:
     # members added in commented lines have a pending membership
     # and will be added back once it is acquired.
     # - adilGhaffarDev
+    - aniruddha2000
     - cahillsf
+    - cecilerobertmichon
     - chiukapoor
+    - chrischdi
     - furkatgofurov7
     - g-gaston
     - hackeramitkumar
+    - joekr
+    - johannesfrey
+    - killianmuldoon
     # - kranurag7
     # - mjlshen
     - nawazkh
     - nprokopic
     - Prajyot-Parab
     # - razashahid107
+    - sbueringer
     # - Sunnatillo
+    - tobiasgiese
```

Comment:
This is a legit removal. Ref: https://github.com/kubernetes/org/pull/4353

Reply:
You can drop these changes since they are supposed to be removed.

Reply:
+1

Reply:
Note: Changes to the whole file can be dropped. GitHub UI is weird and doesn't show the whole selection.

Reply:
dropped the respective commit. Thanks!